addons/crm/models/crm_team.py

```diff
@@ -657,6 +693,7 @@ def _assign_and_convert_leads(self, force_quota=False):
         _logger.info('Assigned %s leads to %s salesmen', sum(len(r['assigned']) for r in result_data.values()), len(result_data))
         for member, member_info in result_data.items():
             _logger.info('-> member %s of team %s: assigned %d/%d leads (%s)', member.id, member.crm_team_id.id, len(member_info["assigned"]), member_info["quota"], member_info["assigned"])
+        _logger.info('%s preferred leads have not been assigned to preference because of quota on a total of %s leads with preference', total_preffered_count - total_assigned_preffered_count, total_preffered_count)
```

Comment:
It would more interesting to have this information per team. Could be logged directly in the loop I think